modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine_pools_hcp_{context}"]
+= Machine pools in ROSA with Hosted Control Plane (HCP) clusters
+
+In ROSA with HCP clusters, the hosted control plane always spans all three of the Availability Zones (AZ) in the installed cloud region. However, unlike ROSA Classic clusters, machine pools in ROSA with HCP clusters do not span AZs. This simplifies the process of configuring machine pools as each machine pool will only require one subnet for the given AZ. Users can also operate a machine pool in a single AZ for testing without incurring the costs associated with cross-AZ traffic. During the ROSA with HCP installation, a single machine pool with one worker node is created for each AZ by default. You can customize the machine pool deployments in your ROSA with HCP cluster according to your organization's requirements.
```

Comment:
I'd suggest cutting down the words on this. 

In ROSA with HCP clusters, the hosted control plane always spans all three of the Availability Zones (AZ) in the installed cloud region. Each machine pool in a HCP cluster is deployed in a single subnet within a single AZ. 

The other statements mentioned are not correct. By writing more, we increase the chances of errors.